.github/workflows/gobump.yml

```diff
@@ -14,6 +14,7 @@ jobs:
     steps:
       - name: Update go.mod and open a PR
         env:
+          GITHUB_TOKEN: ${{ secrets.SCHUTZBOT_GITHUB_ACCESS_TOKEN }}
           GH_TOKEN: ${{ secrets.SCHUTZBOT_GITHUB_ACCESS_TOKEN }}
```

Comment:
Why define it twice?  If `gh` requires `GH_TOKEN`, you cam just change the push command to `git push -f https://"$GH_TOKEN"@github.com/schutzbot/images.git`

Reply:
Genius! Only three coffees in and one coke, but on my defence I also had one 🍺  Rebased.